spec/decision_service_spec.rb

```diff
@@ -611,6 +591,47 @@
         expect(spy_logger).to have_received(:log).once
                                                  .with(Logger::DEBUG, "User '#{user_id}' does not meet the conditions to be in rollout rule for audience '#{audience_name}'.")
       end
+
+      it 'should not bucket the user into the "Everyone Else" rule' do
```

Comment:
Make it 
 it 'should not bucket the user into the "Everyone Else" rule when audience mismatch'